docs/source/io_formats/mgxs_library.rst

```diff
@@ -8,297 +8,155 @@ OpenMC can be run in continuous-energy mode or multi-group mode, provided the
+               or number of points (depending on the value of `scatter-format`)
+               used to describe the angular distribution associated with each
+               group-to-group transfer probability.
+
+**/<library name>/kTs/**
+
+:Datasets: - **<TTT>K** (*double*) -- kT values (in MeV) for each Temperature
+             TTT (in Kelvin), rounded to the nearest integer
+
+**/<library name>/<TTT>K/**
+
+Temperature-dependent data, provided for temperature <TTT>K.
+
+:Datasets: - **total** (*double[]* or *double[][][]*) -- Total cross section.
+             This is a 1-D vector if `representation` is "isotropic", or a 3-D
+             vector if `representation` is "angle" with dimensions of
+             [groups, azimuthal, polar].
+           - **absorption** (*double[]* or *double[][][]*) -- Absorption
+             cross section.
+             This is a 1-D vector if `representation` is "isotropic", or a 3-D
+             vector if `representation` is "angle" with dimensions of
+             [groups, azimuthal, polar].
+           - **fission** (*double[]* or *double[][][]*) -- Fission
+             cross section.
+             This is a 1-D vector if `representation` is "isotropic", or a 3-D
+             vector if `representation` is "angle" with dimensions of
+             [groups, azimuthal, polar].  This is only required if the data set
+             is fissionable and fission-tallies are expected to be used.
+           - **kappa-fission** (*double[]* or *double[][][]*) -- Kappa-Fission
+             (energy-release from fission) cross section.
+             This is a 1-D vector if `representation` is "isotropic", or a 3-D
+             vector if `representation` is "angle" with dimensions of
+             [groups, azimuthal, polar].  This is only required if the data set
+             is fissionable and fission-tallies are expected to be used.
+           - **chi** (*double[]* or *double[][][]*) -- Fission neutron energy
+             spectra.
+             This is a 1-D vector if `representation` is "isotropic", or a 3-D
+             vector if `representation` is "angle" with dimensions of
+             [groups, azimuthal, polar].  This is only required if the data set
+             is fissionable and fission-tallies are expected to be used.
+           - **nu-fission** (*double[]* to *double[][][][]*) -- Nu-Fission
+             cross section.
+             If **chi** is provided, then `nu-fission` has the same
+             dimensionality as `fission`.  If **chi** is not provided, then
+             the `nu-fission` data must represent the fission neutron energy
+             spectra as well and thus will have one additional dimension
+             for the outgoing energy group.  In this case, `nu-fission` has the
+             same dimensionality as `multiplicity matrix`.
+           - **inverse velocities** (*double[]*) -- Average inverse velocity
```

Comment:
Could this also be changed to `inverse-velocity` to have consistency with the tally score and MGXS object?